dangerfile.ts

```diff
@@ -41,18 +40,6 @@ if (pr.base.repo.full_name !== pr.head.repo.full_name) {
   warn("This PR comes from a fork, and won't get JS generated for QA testing this PR inside the Emission Example app.")
 }
 
-// When there are app-changes and it's not a PR marked as trivial, expect
-// there to be CHANGELOG changes.
-const changelogChanges = includes(modified, "CHANGELOG.md")
-if (modifiedAppFiles.length > 0 && !trivialPR && !changelogChanges) {
-  fail("No CHANGELOG added.")
-}
-
-// Check that the changelog contains only H3's
-if (changelogChanges && fs.readFileSync("CHANGELOG.md", "utf8").includes("\n## ")) {
-  fail("Changelog must only contain H3's, but contains H2's.")
-}
-
```

Comment:
~Was this removed to avoid warning on update PRs? We can check the PR metadata for its author, or to see if it was a bot, to avoid these. I'd prefer to keep them around, though I'm just now realizing that I forgot to discuss the Emission changelog during this mornings Front-End iOS Practice meeting.~ I see, it's generated for us. Neat!